frontend/packages/dev-console/src/components/import/import-types.ts

```diff
@@ -16,6 +16,10 @@ export type ImageStreamState = {
   hasCreateAccess: ImageStreamPayload;
   selectedImageStream: ImageStreamPayload;
 };
+export interface ImageStreamNsDropdownProps {
```

Comment:
Component props should be co-located in the component file. Although I don't think you need to add props to fix this.